src/main/webapp/dev/DatePickerJsTest.js

```diff
@@ -0,0 +1,30 @@
+QUnit.module('datepicker.js');
+
+QUnit.test('getMinDateForEndDate(startDate)', function(assert) {
+    assert.deepEqual(getMinDateForEndDate(new Date('02/25/2014')), new Date('02/25/2014'), 'Return startDate');
+});
+
+QUnit.test('getMaxDateForStartDate(endDate)', function(assert) {
+    assert.deepEqual(getMaxDateForStartDate(new Date('02/04/2014')), new Date('02/04/2014'), 'Return endDate');
+});
+
+QUnit.test('getMinDateForPublishDate(visibleDate)', function(assert) {
+    assert.deepEqual(getMinDateForPublishDate(new Date('10/13/2014')), new Date('10/13/2014'), 'Return visibleDate');
```

Comment:
Actually, `getMinDateForEndDate` `getMaxDateForStartDate` are not even used in the project. `getMinDateForPublishDate` is used in the project. I thought it is a kind of abstraction although it just returns the parameter.